### PR TITLE
improve topic

### PR DESCRIPTION
**Pull request by mairaw**
Since this topic will be linked from the docs, I did an edit pass on it.

/cc @janvorli

### CODE REVIEW

File: Documentation/self-contained-linux-apps.md [hunk 1]
```diff
@@ -1,29 +1,35 @@
...
 ### Pick the oldest supported distro the application should run at as the source for the dependencies
-Select a distro from which you'll get the dependencies that will be packaged with the application. Using the oldest distro ensures that the 3rd party dependencies won't be referencing functions from libc / libstdc++ that are present in newer versions of those libraries only and that are not present on the supported distros.
-### Get the transitive closure of all the first level dependencies
-That means the dependencies of the first level dependencies, their dependencies, etc. To get that, use the `ldd` command line tool. As a parameter, pass it the root set of dependencies of .NET core listed below. Then add all the dependencies listed to your final list and run `ldd` again on these added ones.
+Select a distro to get the dependencies to be packaged with the application. Using the oldest distro ensures that the third-party dependencies don't reference functions from libc / libstdc++ that are only present in newer versions of those libraries and not present on the supported distros.
```
Comment (karelz) [inline Documentation/self-contained-linux-apps.md]: Nit: on the supported distros -> **in** the suppoted distros

File: Documentation/self-contained-linux-apps.md [hunk 2]
```diff
@@ -1,29 +1,35 @@
...
+### Get the transitive closure of all the first-level dependencies
+That means the dependencies of the first-level dependencies, their dependencies, etc. To get those transitives closures, use the `ldd` command-line tool. As a parameter, pass the root set of .NET Core dependencies to the command as follows:
 * libcoreclr.so
-* System.IO.Compression.Native.so if  your app uses System.IO.Compression.dll assembly
-* System.Security.Cryptography.Native.OpenSsl.so if your app uses System.Security.Cryptography.dll assembly
-* libssl.so.{version} and libcrypto.so.{version} if your app uses System.Security.Cryptography.dll assembly. The {version} depends on the source distro. For Fedora based distros, it is `10`, for all others except for Debian 9 based distros is it `1.0.0` and for Debian 9 based distros it is `1.0.2`. 
-* System.Net.Http.Native.so if your app uses System.Net.Http.dll
+* If your app uses System.IO.Compression.dll assembly: `System.IO.Compression.Native.so`
+* If your app uses the System.Security.Cryptography.dll assembly: `System.Security.Cryptography.Native.OpenSsl.so`, `libssl.so.{version}`, and `libcrypto.so`.
```
Comment (karelz) [inline Documentation/self-contained-linux-apps.md]: libcrypto.so lost the ".{version}" suffix. Was that intentional?

Comment (mairaw) [reply Documentation/self-contained-linux-apps.md]: Good catch, It wasn't intentional.

**Comment by mairaw** [comment]
@karelz I made the changes you suggested. Please take a look. 
@janvorli can you please review this? I wanna make sure I didn't introduce any errors.

**Comment by karelz** [comment]
Ooops, the notification got buried in all repo notifications folder, sorry.
@janvorli can you please check it as well?

**Comment by Petermarcu** [comment]
I think it looks fine. Merging so this doesn't get stale. We can always make more improvements if there are still issues.

**Pull request merged**